clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -54,6 +58,17 @@ object ClioWebClient {
       extends RuntimeException(
         s"Got an error from the Clio server. Status code: $statusCode. Entity: $entityBody"
       )
+
+  def jsonWithDocumentStatus(input: Json): Json = {
```

Comment:
Not sure it makes sense for `ClioWebClient` to own this method, `Metadata` might be a better spot.

Reply:
Also, should this take a `DocumentStatus` as input? If not, I'd rename to be clear it's setting status = Normal